flask_resty/spec/spec_declaration.py

```diff
@@ -0,0 +1,9 @@
+class SpecDeclaration(object):
+
+    def update_spec(self, path, spec, **kwargs):
```

Comment:
Actually I guess not - since we do `self.whatever` everywhere, the relevant attributes might only be available on instances.